extensions/blocks/eventbrite/eventbrite.php

```diff
@@ -47,14 +75,62 @@ function render_block( $attr, $content ) {
 	);
 
 	$widget_id = wp_unique_id( 'eventbrite-widget-' );
-
-	wp_enqueue_script( 'eventbrite-widget', 'https://www.eventbrite.com/static/widgets/eb_widgets.js', array(), JETPACK__VERSION, true );
-
-	// Add CSS to hide direct link.
-	Jetpack_Gutenberg::load_assets_as_required( FEATURE_NAME );
+	$is_amp    = is_amp_request();
 
 	// Show the embedded version.
 	if ( empty( $attr['useModal'] ) && ( empty( $attr['style'] ) || 'modal' !== $attr['style'] ) ) {
+		return render_embed_block( $widget_id, $is_amp, $attr );
+	} else {
+		return render_modal_block( $widget_id, $is_amp, $attr, $content );
+	}
+}
+
+/**
+ * Render block with embed style.
+ *
+ * @param string $widget_id Widget ID to use.
+ * @param bool   $is_amp    Whether AMP page.
+ * @param array  $attr      Eventbrite block attributes.
+ * @return string Rendered block.
+ */
+function render_embed_block( $widget_id, $is_amp, $attr ) {
+
+	// $content contains a fallback link to the event that's saved in the post_content.
+	// Append a div that will hold the iframe embed created by the Eventbrite widget.js.
+	$classes = Jetpack_Gutenberg::block_classes( FEATURE_NAME, $attr );
+
+	$classes .= ' wp-block-jetpack-eventbrite--embed';
+
+	$direct_link = sprintf(
+		'<a href="%s" rel="noopener noreferrer" target="_blank" class="eventbrite__direct-link" %s>%s</a>',
+		esc_url( $attr['url'] ),
+		$is_amp ? 'placeholder fallback' : '',
+		esc_html__( 'Register on Eventbrite', 'jetpack' )
+	);
+
+	if ( $is_amp ) {
+		$embed = sprintf(
+			'<amp-iframe src="%s" layout="responsive" resizable width="1" height="1" sandbox="allow-scripts allow-same-origin allow-forms"><button overflow>%s</button>%s</amp-iframe>',
+			esc_url(
+				add_query_arg(
+					array(
+						'eid'    => $attr['eventId'],
+						'parent' => rawurlencode( get_current_url() ),
+					),
+					'https://www.eventbrite.com/checkout-external'
+				)
+			),
+			esc_html__( 'Expand', 'jetpack' ),
+			$direct_link
+		);
+	} else {
+		$embed = $direct_link;
+
+		wp_enqueue_script( 'eventbrite-widget', 'https://www.eventbrite.com/static/widgets/eb_widgets.js', array(), JETPACK__VERSION, true );
+
+		// Add CSS to hide direct link.
+		Jetpack_Gutenberg::load_assets_as_required( FEATURE_NAME );
```

Comment:
I think we may need it in the AMP view as well. Right now here is what I see when clicking on the button in the AMP view:

![image](https://user-images.githubusercontent.com/426388/93470173-c5f20200-f8f1-11ea-8057-5c8969cca359.png)

Reply:
@jeherve What theme are you testing this with? I'm checking with Twenty Twenty and I see this:

Modal Not Opened | Model Opened
--------------------|---------------
![image](https://user-images.githubusercontent.com/134745/93704610-d6120900-facc-11ea-93c6-6ce775dadc44.png) | ![image](https://user-images.githubusercontent.com/134745/93704790-f3df6e00-facc-11ea-9034-85f189246ceb.png)

Can you share your `post_content` that you tested with as well?

Reply:
When I "Validate URL" via the AMP plugin, I can see that `view.css` is being added to the page:

![image](https://user-images.githubusercontent.com/134745/93721082-63437500-fb42-11ea-9b2b-8841bb735658.png)

(Disregard that AMP is incorrectly being attributed as the plugin which is adding the stylesheet to the page.)

Reply:
Hm, I can't reproduce anymore, so I guess we're good :)